index.html

```diff
@@ -67,7 +68,219 @@ <h1 class="header-text">
+                        css
+                    </button>
+                    <button id="r" type="submit">
+                        javaScript
+                    </button>
+                </section>
+
+                <section class="prompt">
+                    <button type="submit">See Project</button>
+                </section>
+            </div>
+        </div>
+    </div>
+    <div class="about">
+        <h1 class="header-text">
+            About Myself
+        </h1>
+
+        <p class="text">
+            Hello I’m a software developer! I can help you build a product , feature or website Look through some of
+            my work and experience! If you like what you see and have a project you need coded, don’t hestiate to
+            contact me.
+        </p>
+        <p class="connect">
+            LET’S CONNECT
+        </p>
+        <ul class="socialmedia">
+            <li><img src="./images/git.svg" alt="git"> </li>
+            <li> <img src="./images/linkedin.svg" alt="linkedin"> </li>
+            <li> <img src="./images/tumblur.svg" alt="tumblur"> </li>
+            <li> <img src="./images/twitter.svg" alt="twitter"> </li>
+            <li><img src="./images/medium.svg" alt="medium"> </li>
+        </ul>
+        <section class="prompt">
+            <button type="submit">Get my resume</button>
+        </section>
+        <div class="languages">
+            <section class="lang-drop">
+                <ul>
+                    <li>
+                        <div class="lang-head">
+                            <p>Languages</p>
+                            <img src="./images/down.png" alt="dropdown">
+                        </div>
+                        <div id="buttons">
+                            <button type="button">
+                                <img src="./images/js.svg" alt="js">
+                                javaScript
+
+                            </button>
+                            <button type="button">
+                                <img src="./images/html.svg" alt="html">
+                                HTMl
+                            </button>
+                            <button type="button">
+                                <img src="./images/css.svg" alt="css">
+                                css
+                            </button>
+                        </div>
+                    </li>
+                    <li>
+                        <div class="framework">
+                            <p>Framework</p>
+                            <img src="./images/right-arrow.svg" alt="right">
+                        </div>
+                    </li>
+                    <li>
+                        <div class="skills">
+                            <p>Skills</p>
+                            <img src="./images/right-arrow.svg" alt="right">
+                        </div>
+                    </li>
+                </ul>
+
+
+
+            </section>
```

Comment:
- [ ] Kindly ensure that your design for this section matches that of the original design(fonts, margins, padding etc)
- [ ] The button also does not match the original design. Kindly adjust

### Your Design
![image](https://user-images.githubusercontent.com/43995830/153583677-3a685e5a-111b-4bd9-aeda-206bb6f1c91c.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/153583598-5d099108-161a-4c8c-ae85-02d353d189b6.png)